ssz_serialization/merkleization.nim

```diff
@@ -141,6 +145,14 @@ func digest(a, b: openArray[byte]): Digest {.noinit.} =
       # Adjacent in memory, make a single call (avoids copies inside the
       # digester)
       result = digest(makeOpenArray(baseAddr a, a.len + b.len))
+    elif a.len + b.len == 64:
+      # Single call to digester
+      var buf {.noinit.}: array[64, byte]
+      if a.len > 0:
+        copyMem(addr buf[0], unsafeAddr a[0], a.len)
+      # b.len > 0 per above..
+      copyMem(addr buf[a.len], unsafeAddr b[0], b.len)
+      result = digest(buf)
```

Comment:
Ideally, this is at some point fixed in upstream Nim, rather than requiring this kind of ad-hoc workaround

Reply:
https://github.com/nim-lang/Nim/issues/19357 (highlighted in initial section of this  file)